src/hooks/useLightwellRouteSetup.ts

```
@@ -0,0 +1,45 @@
+import { useLayoutEffect } from 'react';
+import { useSetAtom } from 'jotai';
+import { layoutForceGlassThemeAtom } from '../state/atoms/releaseAtom';
+import { LIGHTWELL_PATH } from '../utils/common';
+
+const FELT_THEME_CLASS = 'pf-v6-theme-felt';
+const GLASS_THEME_CLASS = 'pf-v6-theme-glass';
+
+const isLightwellRoute =
+  window.location.pathname === LIGHTWELL_PATH || window.location.pathname.startsWith(`${LIGHTWELL_PATH}/`);
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)